variables.tf

```diff
@@ -59,6 +59,21 @@ variable "logdna_add_cluster_name" {
   default     = true
 }
 
+# LogDNA agent custom settings to setup Kubernetes metadata logs filtering
+# Ref https://github.com/logdna/logdna-agent-v2/blob/3.8/docs/KUBERNETES.md#configuration-for-kubernetes-metadata-filtering
+
+variable "logdna_agent_custom_line_inclusion" {
+  description = "LogDNA agent custom configuration for line inclusion setting LOGDNA_K8S_METADATA_LINE_INCLUSION."
+  type        = string
+  default     = null
+}
+
+variable "logdna_agent_custom_line_exclusion" {
+  description = "LogDNA agent custom configuration for line exclusion setting LOGDNA_K8S_METADATA_LINE_EXCLUSION."
+  type        = string
+  default     = null
+}
```

Comment:
remove these variables, and add the code directly in main.tf of the example

Reply:
one downside to do this: if I hardcode the values in the main.tf body of the helm_release I lose the option to set these two configurations only if the filter configuration is needed (using dynamic block)

Unless these variables are removed and the configuration is hardcoded into locals block, this would allow to keep the dynamic configuration

Reply:
oh, I think I left the comment in the wrong place. I meant to hard code values in main.tf of the **example**. Not the main module obviously

Reply:
yea sorry I meant to leave the comment [here](https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/pull/226#discussion_r1400466727)